attitude_fw/ecl_yaw_controller.cpp

```diff
@@ -108,27 +89,36 @@ float ECL_YawController::control_attitude_impl_openloop(const struct ECL_Control
 		return _rate_setpoint;
 	}
 
-//	static int counter = 0;
-	/* Calculate desired yaw rate from coordinated turn constraint / (no side forces) */
-	_rate_setpoint = 0.0f;
+	float constrained_roll;
+	bool inverted = false;
+
+	/* roll is used as feedforward term and inverted flight needs to be considered */
+	if (fabsf(ctl_data.roll) < math::radians(90.0f)) {
+		/* not inverted, but numerically still potentially close to infinity */
+		constrained_roll = math::constrain(ctl_data.roll, math::radians(-80.0f), math::radians(80.0f));
```

Comment:
I think this needs to be limited further for the `tanf(constrained_roll)`, but what makes sense?